cli/commands.go

```diff
@@ -258,6 +261,46 @@ func initDaemon(config *conf.MenderConfig,
 	return daemon, nil
 }
 
+func checkDemoCert() {
+	trustStoreCert, err := os.Open(getLocalTrustMenderCertPath())
+	if err != nil {
+		log.Debugf("Could not open Mender certificate in local trust store: %s", err.Error())
+		return
+	}
+	defer trustStoreCert.Close()
+
+	demoCert, err := os.Open(getMenderDemoCertPath())
```

Comment:
You are calling `getMenderDemoCertPath` several times; shouldn't we create a variable and reuse it?